firebase-sessions/src/test/kotlin/com/google/firebase/sessions/SessionEventTest.kt

```diff
@@ -32,7 +32,8 @@ class SessionEventTest {
     val sessionEvent =
       SessionEvents.startSession(
         FakeFirebaseApp.fakeFirebaseApp(),
-        TestSessionEventData.TEST_SESSION_DETAILS
+        TestSessionEventData.TEST_SESSION_DETAILS,
+        TestSessionEventData.TEST_SESSION_TIMESTAMP_US,
```

Comment:
Any reason why the event timestamp is configurable? Can we leave it as a constant?

Reply:
`startSession` is also used by the SDK. When it leaves off the parameter it uses the current time. Since we can't have the time be different for every test I made it optional to just set the time

Reply:
I think this is fine for now, but we need a proper time solution at some point. We can't just use [go/javatime](http://go/javatime) because it requires Android API level 26+. Maybe we can add another function in `WallClock` and try to abstract this away?